### PR TITLE
Add August MC video to Monthly Calls page

### LINKED ISSUE #2846: Add August Monthly Call video to Monthly Calls page
### Purpose
To add the August monthly call video (when available) to the [USWDS monthly calls page](https://designsystem.digital.gov/about/monthly-calls/).

### References
[Digital.gov August event page](https://digital.gov/event/2024/08/15/uswds-monthly-call-august-2024/)

### Steps to be completed once video is available:

- [x] Create branch to update YML page
- [x] Submit pull request - will be closed by:  #2873 
- [ ] Pull request approval

### PR DESCRIPTION
**Pull request by finekatie**
# Summary

Adding August Monthly Call content to Monthly Calls page 

> [!Important]
> The changelog date will need to be updated before merge.

## Related issue

Closes #2846 
Related to #[6025 ](https://github.com/uswds/uswds/issues/6025)

## Preview link 
[Monthly calls page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-augMCvideo_MCspg/about/monthly-calls/)

## Problem statement

Needed to get Monthly Calls page up to date with August video. 

## Solution

August Monthly Call summary has been done. We just received video and it needs to be added to page. 

## Testing and review

Need someone to look and make sure link to video works.

### CODE REVIEW

**Comment by finekatie** [comment]
Hey @amyleadem , I updated the changelog too in the branch. How do I include that in this PR? Do I? I've never done a changelog addition before.

**Comment by finekatie** [comment]
link to video and changelog with pending date are ready for review.

File: _data/changelogs/about-monthly-calls.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@ title: Monthly calls
 type: documentation
 changelogURL:
 items:
+ - date: 2024-10-00
+    summary: Added August 2024 monthly call video.
+    githubPr: 2846
```
Comment (amyleadem) [inline _data/changelogs/about-monthly-calls.yml:7]: @finekatie Looks like this just is missing a space to get it to build. I also added a few tweaks to emphasize this update just adds the video and added the correct PR number. 

Related: I am not sure that we actually _need_ a changelog for adding just the video, but I am leaning towards its inclusion here just to err on the side of clarity. But I think it is something we can discuss if desired.

```suggestion
  - date: 2024-10-23
    summary: Added video for August 2024 monthly call.
    githubPr: 2873
```

**Review by amyleadem — CHANGES_REQUESTED** [review]

File: _data/changelogs/about-monthly-calls.yml (hunk 1 above)
Comment (finekatie) [reply _data/changelogs/about-monthly-calls.yml:7]: Works for me! Thank you for looking!

**Review by annepetersen — COMMENTED** [review]
LGTM

**Pull request merged**